Exiled.Events/Patches/Events/Scp244/DamagingScp244.cs

```diff
@@ -0,0 +1,112 @@
+// -----------------------------------------------------------------------
+// <copyright file="DamagingScp244.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Scp244
+{
+#pragma warning disable SA1313
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Reflection.Emit;
+
+    using Exiled.API.Features;
+    using Exiled.API.Features.DamageHandlers;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using InventorySystem;
+    using InventorySystem.Items.Usables.Scp244;
+    using InventorySystem.Searching;
+
+    using NorthwoodLib.Pools;
+
+    using PlayerStatsSystem;
+
+    using UnityEngine;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="Scp244DeployablePickup.Damage"/> to add missing logic to the <see cref="Scp244DeployablePickup"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(Scp244DeployablePickup), nameof(Scp244DeployablePickup.Damage))]
+    internal static class DamagingScp244
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Shared.Rent(instructions);
+
+            Label returnFalse = generator.DefineLabel();
+
+            Label continueProcessing = generator.DefineLabel();
+
+            LocalBuilder eventHandler = generator.DeclareLocal(typeof(DamagingScp244EventArgs));
+
+#pragma warning disable SA1118 // Parameter should not span multiple lines
```

Comment:
No, do not disable it. Just stop leaving a blank line after every instruction.

Reply:
![image](https://user-images.githubusercontent.com/24619207/167973729-a915e412-f878-42e7-b699-3a6e3a99b775.png)

Reply:
![image](https://user-images.githubusercontent.com/24619207/167973754-70680c54-b8bc-4fd5-bd2e-0d683c130bbb.png)

Reply:
I mean.. the other Transpilers themselves have the same disabling, spacing does not resolve the type of issue which is multi-line parameter.

Reply:
I can move it up, but going by precedent, it should exist in the file. Alternatively, make it a global configured ignored as multi-line parameter is used for most Transpilers. Up to you

Reply:
Oh lol, move it up then. And yeah, stop leaving blank lines. That's all 👍🏼

Reply:
Done, on @louis1706 to merge.

Reply:
Done ping me discord :) when you want to merge i will see it faster than GitHub